Sources/MixpanelInstance.swift

```diff
@@ -214,7 +214,7 @@ open class MixpanelInstance: CustomDebugStringConvertible, FlushDelegate, AEDele
     #endif // DECIDE
     
     #if !os(OSX) && !os(watchOS)
-    init(apiToken: String?, flushInterval: Double, name: String, optOutTrackingByDefault: Bool = false) {
+    init(apiToken: String?, flushInterval: Double, name: String, optOutTrackingByDefault: Bool = false, trackAutomaticEvents: Bool = false) {
```

Comment:
default is `false` here? but `true` in Mixpanel.swift?

Reply:
should be `true`, good catch